internal/utils_test.go

```diff
@@ -1152,3 +1316,66 @@ func TestVerifyManifestPath(t *testing.T) {
 		)
 	}
 }
+
+func TestProcessKustomizeDir(t *testing.T) {
+	baseDirectory, err := filepath.EvalSymlinks(t.TempDir())
+	if err != nil {
+		t.Fatalf("Failed to evaluate symlinks for the base directory: %v", err)
+	}
+
+	// Set up directory structure, with 'workingdir' as target directory:
+	// baseDirectory (t.TempDir())
+	// └── kustomizedir
+	kustomizeDir := path.Join(baseDirectory, "kustomizedir")
+
+	err = os.Mkdir(kustomizeDir, 0o777)
+	if err != nil {
+		t.Fatalf("Failed to create the directory structure %s: %v", kustomizeDir, err)
+	}
+
+	// Create files in baseDirectory/kustomizedir
+	manifestPaths := map[string]string{
+		"kustomization.yaml": `
+resources:
+- configmap.yaml
+- https://github.com/dhaiducek/policy-generator-plugin/examples/input-kustomize/?ref=support-kustomize
+
+namespace: kustomize-test
+`,
+		"configmap.yaml": `
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-configmap
+data:
+  game.properties: |
+    enemies=goldfish
+`,
+	}
+
+	for filename, content := range manifestPaths {
+		manifestPath := path.Join(kustomizeDir, filename)
+
+		err = ioutil.WriteFile(manifestPath, []byte(content), 0o666)
+		if err != nil {
+			t.Fatalf("Failed to write %s", manifestPath)
+		}
+	}
+
+	t.Run(
```

Comment:
Does this need to run in a `t.Run` since there aren't multiple testcases?

Reply:
Indeed it does not 🙂